src/platform/posix/posix.c

```diff
@@ -9,10 +9,10 @@
 #include <sof/schedule/ll_schedule.h>
 #include <sof/lib/agent.h>
 
-uint8_t posix_hostbox[MAILBOX_HOSTBOX_SIZE];
-uint8_t posix_dspbox[MAILBOX_DSPBOX_SIZE];
-uint8_t posix_stream[MAILBOX_STREAM_SIZE];
-uint8_t posix_trace[MAILBOX_TRACE_SIZE];
+uint8_t posix_hostbox[MAILBOX_HOSTBOX_SIZE] __aligned(8);
```

Comment:
Inline comment would have been nice, but this file's history is simple enough so curious minds will find the explanation in git commit.